docs.html

```diff
@@ -216,14 +216,14 @@ <h2 id=vars>Variables</h2>
 <input class='run' type='button' value='Run'> 
 
 <p> 
-To change the value of the variable use <code>=</code>. In V variables are 
+Use <code>=</code> to change the value of the variable. In V, variables are 
```

Comment:
A general rule: do not use a comma after introductory prepositional phrases which are less than five words long. In this sentence a comma is unneeded and incorrect to use:

In Section 2 we will study cells.